build.ps1

```diff
@@ -70,162 +71,154 @@ function Invoke-Configure {
         Write-Host "Installing Chocolatey"
         Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1')) | out-null
     }
-    else {
-        Write-Host "Making sure Chocolatey is current."
-        choco upgrade chocolatey --confirm | Out-Null
-    }
 
-    # We need the native library dependencies for `hab`
-    # Until we have habitat packages on Windows, there is
-    # a chocolatey package hosted in MyGet with the native
-    # dependencies built.
-    if ((choco list habitat_native_dependencies --local-only) -match '^1 packages installed\.$') {
-        choco upgrade habitat_native_dependencies --confirm -s https://www.myget.org/F/habitat/api/v2  --allowemptychecksums
-    }
-    else {
-        choco install habitat_native_dependencies --confirm -s https://www.myget.org/F/habitat/api/v2  --allowemptychecksums
+    if (!((choco list habitat --local-only) -match '^1 packages installed\.$')) {
+        choco install habitat -y
     }
 
     if(!(Get-Command git -ErrorAction SilentlyContinue)) {
```

Comment:
This is unneeded, we are guaranteed to have the correct version of git in our base images.

Reply:
yeah in the `BuildKite` case that should never be the case. This is here mainly for the clean VM or container scenario.